### PR TITLE
Error in return value should be tested and fix Description Info in e2e test

### PR DESCRIPTION
**Pull request by tanshanshan**
**What this PR does / why we need it**:

1. Error in return value should be tested
2. fix Description Info

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by nikhita** [comment]
@frapposelli thanks for reviewing this so quick! :)

/assign @jsafrane 
for approval

File: test/e2e/storage/vsphere/vsphere_volume_fstype.go [hunk 1]
```diff
@@ -128,6 +128,7 @@ func invokeTestForInvalidFstype(f *framework.Framework, client clientset.Interfa
 	Expect(err).To(HaveOccurred())
 
 	eventList, err := client.CoreV1().Events(namespace).List(metav1.ListOptions{})
+	Expect(err).To(HaveOccurred())
```
[the remark this answers is not in the document]
Comment (jsafrane) [reply test/e2e/storage/vsphere/vsphere_volume_fstype.go]: `Expect(err).To(HaveOccurred())` expects that `List()` fails.  Why List() fails?

[the remark this answers is not in the document]
Comment (tanshanshan) [reply test/e2e/storage/vsphere/vsphere_volume_fstype.go]: I removed this line.
PTAL.

**Pull request merged**